template/template.go

```diff
@@ -20,6 +24,28 @@ func init() {
 // Render parses and executes a template, returning the results
 // in string format.
 func Render(template string, playload *drone.Payload) (string, error) {
+	if strings.HasPrefix(template, "http") {
+		parsedURL, err := url.Parse(template)
+
```

Comment:
No need to parse the url here because it is parsed in the `http.NewRequest` function which is invoked by `http.Get` so we should be all set:
https://golang.org/src/net/http/request.go?s=17718:17790#L536